app/ledger.js

```diff
@@ -32,6 +32,7 @@ const session = electron.session
 const acorn = require('acorn')
 const ledgerBalance = require('ledger-balance')
 const ledgerClient = require('ledger-client')
+const ledgerGeoIP = require('ledger-geoip')
```

Comment:
please remove all providers in https://github.com/brave/ledger-geoip/blob/master/index.js that do not offer HTTPS